app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :likes, dependent: :destroy
+  has_many :comments, dependent: :destroy
+
+  def update_user_posts_counter
```

Comment:
- [ ] [OPTIONAL] I think it would be a great idea to trigger this method using one of the Active Record callbacks so it can run automatically upon saving a post to the database. You did great in the Like and Comment models. 😎️